Modules/Filtering/ImageStatistics/include/itkLabelStatisticsImageFilter.hxx

```diff
@@ -520,12 +520,21 @@ template <typename TImage, typename TLabelImage>
 void
 LabelStatisticsImageFilter<TImage, TLabelImage>::PrintSelf(std::ostream & os, Indent indent) const
 {
+  using namespace print_helper;
+
   Superclass::PrintSelf(os, indent);
 
-  os << indent << "Number of labels: " << m_LabelStatistics.size() << std::endl;
-  os << indent << "Use Histograms: " << m_UseHistograms << std::endl;
-  os << indent << "Histogram Lower Bound: " << m_LowerBound << std::endl;
-  os << indent << "Histogram Upper Bound: " << m_UpperBound << std::endl;
+  os << indent << "LabelStatistics: " << std::endl;
+  for (auto const & pair : m_LabelStatistics)
+  {
+    std::cout << "{" << pair.first << ": " << pair.second << "}" << std::endl;
```

Comment:
std::cout?

Reply:
:100: thanks @issakomi. Fixed in [02bc0f4](https://github.com/InsightSoftwareConsortium/ITK/commit/02bc0f4a2731592eb3b41650dc6fbb01a62829fd).